content/04.study.md

```diff
@@ -281,72 +281,78 @@ Though the results are preliminary and still based on a validation set rather th
+Secondary structure can be represented in three (alpha helix, beta strand, and loop regions; Q3) or eight finer-grained states (Q8) [@doi:10.1002/bip.360221211].
 
-One can represent protein secondary structure with three different states (alpha helix, beta strand, and loop regions) or eight finer-grained states.
-Accuracy of a three-state prediction is called Q3, and accuracy of an 8-state prediction is called Q8.
-Several groups [@doi:10.1371/journal.pone.0032235; @doi:10.1109/TCBB.2014.2343960; @doi:10.1038/srep11476] applied deep learning to protein secondary structure prediction but were unable to achieve significant improvement over the *de facto* standard method PSIPRED [@doi:10.1006/jmbi.1999.3091], which uses two shallow feedforward neural networks.
+Earlier attampts of applying deep learning to secondary structure prediction [@doi:10.1371/journal.pone.0032235; @doi:10.1109/TCBB.2014.2343960; @doi:10.1038/srep11476] didn't achieve significant improvenets over the widly used and relatively simple PSIPRED [@doi:10.1006/jmbi.1999.3091].
 In 2014, Zhou and Troyanskaya demonstrated that they could improve Q8 accuracy by using a deep supervised and convolutional generative stochastic network [@arxiv:1403.1347].
 In 2016 Wang et al. developed a DeepCNF model that improved Q3 and Q8 accuracy as well as prediction of solvent accessibility and disorder regions [@doi:10.1038/srep18962; @doi:10.1007/978-3-319-46227-1_1].
-DeepCNF achieved a higher Q3 accuracy than the standard maintained by PSIPRED for more than 10 years.
-This improvement may be mainly due to the ability of convolutional neural fields to capture long-range sequential information, which is important for beta strand prediction.
-Nevertheless, the improvements in secondary structure prediction from DeepCNF are unlikely to result in a commensurate improvement in tertiary structure prediction since secondary structure mainly reflects coarse-grained local conformation of a protein structure.
+Following these successes, recently proposed methods achieve significantly better results than PSIPRED using Deep Learning architectures, with significantly better DL methods today being NetSurfP-2.0 [@doi:10.1002/prot.25674] and Porter 5 [@doi:10.1038/s41598-019-48786-x].
+
+The improvement obtained by these methods may be mainly due to the ability of convolutional neural fields to capture long-range information.
+Top methods still heavily rely on the creation of profiles from Multiple Sequence Alignments (MSAs).
+Models relying on LMs have yet to reach the accuracy of evolutionary based methods, but are able to deliver results for proteins for which MSAs can't be computed and in general execute at a fracion of the time with respect to evolutionary based approaches [@doi:10.1186/s12859-019-3220-8].
```

Comment:
```suggestion
Models relying on LMs have yet to reach the accuracy of evolutionary based methods even for proteins for which good MSAs can't be built, but these LM methods in general execute at a fraction of the time with respect to evolutionary based approaches [@doi:10.1186/s12859-019-3220-8].
```

Reply:
@j3xugit

I think your addition has not yet been experimentally validated (`even for proteins for which good MSAs cannot be built`), and is, in large part, a repetition of the first part of the sentence (`Models relying on LMs have yet to reach the accuracy of evolutionary based methods`), which is the best (to my knowledge) one can say about the comparison in accuracy between evo vs. LMs.

My original intent (`but are able to deliver results for proteins for which MSAs can't be computed`) was meant in the sense that evo models (at least the ones I know) actually fail to produce results if MSAs can't be built.

If we can't agree, I'm happy to leave out this second part of the sentence. I would also rewrite the second part into its own sentence and make it clearer, here my suggestion:

```suggestion
Models relying on LMs have yet to reach the accuracy of evolutionary based methods.
On the upside, LMs require a fraction of the resources for inference compared to evolutionary based approaches [@doi:10.1186/s12859-019-3220-8].
```

Reply:
It is incorrect to say that evo models fail to produce results if MSAs can't be built.  In fact my deep models (and other similar ones) works well on some proteins for which MSAs cannot be built. Baker group has also shown that the deep models developed by his group (which is similar to mine) works well on a good portion of human-designed proteins although MSAs cannot be built for them.

Reply:
Are there specific references we should have in mind to support the discussion in these sentences?  That might help make sure all of us are looking at the same models and results.

Reply:
Right, I might have been too black and white here :) sorry about that!

My belief is that if a model bases predictions on only PSSMs as inputs (or evo couplings), but for a certain dark protein there simply isn't an MSA to start with, those predictions can't be trusted (I picture this as having an input matrix with all zeroes, just to give an idea). It's more of a "conceptual" point than reality ot things.
Models out there today do all sorts of things, including combining sequence-based features (often learned, e.g. via CNNs) with MSA extracted features, so mine would ba simplification.

I'm happy either way :) The more fundamental point of this sentence, for me, is that we still don't have a clear understanding on how well LMs work on those proteins for which also MSA-based methods perform arguably well, and how much we are buying ourselves coverage by using these models instead (or combinations of the two). But results for that will be out in Dec with CASP, I feel.

Reply:
@sacdallago can you please edit these sentences to take into account @j3xugit's comment and the trRosetta results for de novo proteins?  Or @j3xugit could suggest new text for @sacdallago to review.

https://www.pnas.org/content/117/3/1496 (`doi:10.1073/pnas.1914677117`) could also be a relevant reference to add.

Reply:
If it's ok, I'll edit this the week of Aug 31st (I'm currently on holidays and it's hard to get into the writing headspace -- especially without a laptop & context overview)

Reply:
Sending a brief reminder about these edits @sacdallago.  It shouldn't require anything too extensive.

Reply:
@agitter 

1. Thanks very much for the reminder -- unfortunately I had to attach 1 week leave of absence to my holidays due to some unfortunate family circumstances. AKA: sorry for the delay
2. I don't quite know how to integrate the TrRosetta paper in exactly this section. TrRosetta doesn't use language models to extract additional features, it's still relies exclusively on MSAs (from my understanding). I did add it to an earlier sentence where it appeared to make more sense.
3. I updated some other sections, which in the meantime have seen some new pre-prints and work.

I'll update the PR in about 10 min with the latest changes